src/Turbo/README.md

```diff
@@ -0,0 +1,267 @@
+    "message_1" in the list will be displayed again.
+    -->
+    <a href="/messages">Back</a>
+
+<!-- End the frame -->
+{{ turbo_frame_end() }}
+</body>
+```
+
+### 3. Create real-time applications with Turbo Streams
+
+[Turbo Streams](https://turbo.hotwire.dev/handbook/streams) are a way to update the content
+of HTML pages in real-time (server push) with no JavaScript coding required.
+
+Turbo Streams work by relying on a transport (a way to push messages from the server to
+the client in real-time), like websockets or [Symfony Mercure](https://symfony.com/doc/current/mercure.html).
+
+Then, Turbo Streams defines a protocol to **describe changes to apply to a page using HTML**.
+For instance, the following snippet of code could be sent over the real-time transport:
+
+```html
+<turbo-stream action="append" target="messages">
+    <template>
+        <div id="message_1">This div will be appended to the element with the DOM ID "messages".</div>
+    </template>
+</turbo-stream>
+```
+
+When this code will be received by the client, Turbo will apply the change to the page. In this example,
+a div will be appended to the DOM element with ID "messages".
+
+There are 5 native actions available in Turbo Streams: **append**, **prepend**, **replace**,
+**update**, and **remove**. In addition to these 5 actions, you can add custom behavior by using
+[Stimulus](https://stimulus.hotwire.dev/), which is native to Symfony UX.
+
+Symfony UX Turbo helps setting up Turbo Streams by providing native adapters for transports (like
+Mercure) and by providing helpers to create streams.
+
+**Example: a chat**
+
+For instance, let's imagine we want to build a simple chat system: we want to display messages in real-time
+to all clients currently connected when a message is posted. To do so, we can leverage Turbo Streams.
+
+First, we need to configure a transport. In the Symfony ecosystem, we recommend to use
+[Mercure](https://symfony.com/doc/current/mercure.html), a protocol based on Server-Sent Events which
+provides many very useful features on top of them (authentication, security, topics, ...).
+
+To configure it, [install Mercure](https://symfony.com/doc/current/mercure.html) and configure UX Turbo
+to use it:
+
+```yaml
+# config/packages/turbo.yaml
+turbo:
+    streams:
+        adapter: turbo.streams.adapter.mercure
+        options:
+            hub: '%env(MERCURE_PUBLISH_URL)%'
+```
+
+Then, let's build the chat view and configure it to listen to the "chat" topic on Mercure:
+
+```php
+class ChatController extends AbstractController
+{
+    /**
+     * @Route("/chat", name="chat")
+     */
+    public function chat(): Response
+    {
+        return $this->render('chat.html.twig');
+    }
+}
+```
+
+```twig
+{# chat.html.twig #}
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    {# This line configures Turbo to listen to the "chat" topic on Mercure #}
+    {{ turbo_stream_from('chat') }}
```

Comment:
Though I appreciate you technically want it to live within a `block` in here, semantically it would make greater sense to register it within your Twig file like that:

```twig
{% listen_turbostream "chat" %}
```